docs/developer/discounts/vouchers.mdx

```diff
@@ -42,6 +42,22 @@ The voucher can be also set as single-use, when `singleUse` flag is set to `true
 
 Those limitation options can be combined. For example, if `usageLimit` is set to `10` and `applyOncePerCustomer` is set to `true`, the voucher can be used by the first 10 users.
 
+### Voucher usage in draft orders
+
+Since version 3.18, Saleor is able to calculate voucher usage in draft orders. To turn on such a behavior, `includeDraftOrderInVoucherUsage` flag for given `Channel` must be set to `true`. Use [channelUpdate](api-reference/channels/mutations/channel-update.mdx) mutation to switch the flag.
```

Comment:
```suggestion
Since version 3.18, Saleor is able to calculate voucher usage in draft orders. To turn on such a behavior, `includeDraftOrderInVoucherUsage` flag for the given `Channel` must be set to `true`. Use [channelUpdate](api-reference/channels/mutations/channel-update.mdx) mutation to switch the flag.
```